src/main/resources/css/layout.css

```diff
@@ -0,0 +1,1073 @@
+/************************************************************************
+-) BASIC CONTAINERS
+*************************************************************************/
+
+body {
+	font-size:13px;
+	font-family: Arial, Helevtica, Verdana, san-serif;
+	color: #777777;
+}
+
+a {
+	text-decoration: none;
+}
+
+#fullwidth_header {
+	width: 100%;
+	height: 80px;
+	background: url(header_bg.png) repeat-x;
+	position: fixed;
+	top: 0;
+	z-index: 9999;
+}
+
+#fullwidth_gradient {
+	width: 100%;
+	height: 180px;
+	background: url(gradient_bg.png) repeat-x;
+	margin-top: 80px;
+}
+
+.iframe {
+	z-index: 1px;
+}
+
+
+/************************************************************************
+-) HEADER
+*************************************************************************/
+
+
+.heading {
+	margin-top: 40px;
+}
+
+.heading h1 {
+	margin-bottom: 20px;
+}
+
+.subhead {
+	display: block;
+	font-family: Helvetica;
+	font-size: 14px;
+	font-weight: bold;
+	color: #555555;
+}
+
+
+#monitor {
+	height: 298px;
+	background: url(monitor.png) no-repeat;
+}
+
+#cycle {
+	width:322px;
+	height:185px;
+	position:absolute;
+	z-index:1;
+	margin:22px 0 0 45px;
+}
+
+.heading, .slider {
+	margin-bottom: 20px;
+}
+
+
+/*** LOGO ***/
+
+.logo, .logo a {
+	display:block;
+	height:70px;
+	width:320px;
+	position:relative;
+	left:0px;
+	top:7px;
+	border: none;
+	padding: 0;
+	margin:0;
+}
+
+.logo a, .logo a:hover {
+	top:0;
+	left:0;
+	text-indent:-9999px;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Negative text-indent doesn't work well with RTL. If you use text-indent for image replacement explicitly set direction for that item to ltr.](https://www.codacy.com/app/damianszczepanik/cucumber-reporting/file/2139367645/issues/source?bid=3059308&fileBranchId=3150237#l93)